dcos/dcos-ui
# feat(ToggleValue): add new toggle value component

**Description**:
Add a `ToggleValue` component. It has an API like this:

```
<ToggleValue 
    primaryValue="5 days overdue" 
    secondaryValue="Feb 20, 2017" 
/>
```

This component will initially be used for the node licensing expiry row. In the future, it's intended to be rolled out as a general UX pattern when displaying dates/times anywhere in DC/OS UI (currently Marathon UI has this). 

**Testing**:
1. Checkout this branch.
2. In private plugins, check out the branch in this [PR](https://github.com/mesosphere/dcos-ui-plugins-private/pull/648).
3. Navigate to the `/overview` page and see the toggle on the row called "License Expiry"

**Checklist**
